src/map/packets/independant_animation.cpp

```diff
@@ -0,0 +1,25 @@
+﻿#include "../../common/socket.h"
+
+#include "../entities/baseentity.h"
+
+#include "independant_animation.h"
+
+CIndependantAnimationPacket::CIndependantAnimationPacket(CBaseEntity* PEntity, uint16 animId, uint8 mode, uint8 unknown)
+{
+	this->type = 0x3A;
+	this->size = 0x14;
+
+	if (PEntity)
+	{
+        ref<uint32>(0x04) = PEntity->id;
+        ref<uint32>(0x08) = PEntity->id;
+
+        ref<uint16>(0x0C) = PEntity->targid;
+        ref<uint16>(0x0E) = PEntity->targid;
```

Comment:
Looks like this should take a target and pass that as `0x08` and `0x0E`. See [Windower docs](https://github.com/Windower/Lua/blob/308ec1d44e94250477db634a8a0552d7e687a88a/addons/libs/packets/fields.lua#L2123-L2131).